frontend/assets/styles/sass/main.scss

```diff
@@ -244,7 +244,8 @@
         word-wrap: break-word;
         word-break: break-all;
         overflow: hidden;
-        font: Bold 16px/20px Proxima Nova;
```

Comment:
This code is no longer necessary, since the font is already imported at the parent element. Also, `16px/20px` does not work, it just gets the `16px`.